lib/cc/analyzer/issue_location_existence_validation.rb

```diff
@@ -0,0 +1,56 @@
+module CC
+  module Analyzer
+    class IssueLocationExistenceValidation < Validation
+      def valid?
+        if location["lines"]
+          valid_lines?
+        elsif location["positions"]
```

Comment:
Should there be an `else` after this to add an error in the event that the location is completely invalid and doesn't have either? Is that covered anywhere else?